tests/test_gdist.py

```diff
@@ -105,3 +105,21 @@ def test_hedgehog_mesh(self):
         # test if the obtained matrix is symmetric
         assert (abs(distances - distances.T) > epsilon).nnz == 0
         assert np.max(distances) <= 1.45
+
+
+class TestDistanceMatrixOfSelectedPoints:
+    def test_flat_triangular_mesh(self):
+        data = np.loadtxt("data/flat_triangular_mesh.txt", skiprows=1)
+        vertices = data[0:121].astype(np.float64)
+        triangles = data[121:].astype(np.int32)
+        points = np.array([2, 5, 10], dtype=np.int32)
+        distances = gdist.distance_matrix_of_selected_points(
+            vertices,
+            triangles,
+            points,
+        )
+        epsilon = 1e-6
+        expected = np.loadtxt("data/flat_triangular_mesh_pairwise_matrix.txt")
```

Comment:
Could this unit-test be improved a bit:
- assert that the result is a sparse matrix
- assert that the number of rows/columns not empty related to the `points` size
?